module_analysis/tests/test_module.py

```diff
@@ -2,11 +2,11 @@
 # @author: Sylvain LE GAL (https://twitter.com/legalsylvain)
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 
-from odoo.tests.common import TransactionCase, at_install, post_install
+from odoo.tests import tagged
+from odoo.tests.common import TransactionCase
 
 
-@at_install(False)
-@post_install(True)
+@tagged("post_install")
```

Comment:
AFAIK, this should be:

```suggestion
@tagged("-at_install", "post_install")
```